api-reference/beta/api/application_post_extensionproperties.md

```diff
@@ -2,7 +2,7 @@
 
 Use this API to create a new extensionProperty.
```

Comment:
Not even sure why this was already published.  We don't support this operation AFAIK on Microsoft Graph.  
For schema extensions we want folks using the new mechanism.